test/api-test.js

```diff
@@ -270,6 +270,34 @@ describe('RippleAPI', function () {
       }, /dropsToXrp:\ invalid\ value\ '\.\.\.'\,\ should\ be\ a\ number\ matching\ \(\^\-\?\[0\-9\]\*\.\?\[0\-9\]\*\$\)\./)
     })
   })
+
+  describe('deriveKeypair', function () {
+    it('returns keypair for secret', function () {
+      var keypair = this.api.deriveKeypair("snsakdSrZSLkYpCXxfRkS4Sh96PMK");
+      assert(keypair.privateKey == "008850736302221AFD59FF9CA1A29D4975F491D726249302EE48A3078A8934D335");
+      assert(keypair.publicKey == "035332FBA71D705BD5D97014A833BE2BBB25BEFCD3506198E14AFEA241B98C2D06");
```

Comment:
Use assert.equal so that failures are reported better:
```
      assert.equal(keypair.privateKey, '008850736302221AFD59FF9CA1A29D4975F491D726249302EE48A3078A8934D335');
      assert.equal(keypair.publicKey, '035332FBA71D705BD5D97014A833BE2BBB25BEFCD3506198E14AFEA241B98C2D06');
```